packages/autocomplete-core/src/utils/createCancelablePromiseList.ts

```diff
@@ -0,0 +1,31 @@
+import CancelablePromise from 'cancelable-promise';
```

Comment:
we should inline this dependency and use only the parts we need IMO, core behaviour shouldn't live externally

Reply:
It's a work in progress, not meant for review yet, and currently messy 🙂